lib/fuel-rest.js

```diff
@@ -73,7 +74,15 @@ class FuelRest {
 		});
 	}
 	_processRequest(options, callback) {
-		this.AuthClient.getAccessToken(clone(options.auth))
+
+		const requestOptions = clone(options.auth) || {};
+
+        if (this.proxy) {
+            requestOptions.proxy = this.proxy;
+            requestOptions.rejectUnauthorized = false;
```

Comment:
Is there a reason that this is passed by default? it's technically enabling a security hole if people don't look at this option in-depth